src/js/plugins/networkcontrol.js

```diff
@@ -46,44 +46,46 @@ class NetworkControl extends Plugin {
             <select id="NetworkInterface">
             </select>
           </div>
-          <div class="label grid__col grid__col--2-of-8">
-            IP Address
-          </div>
-          <div id="ip" class="text grid__col grid__col--6-of-8">
-            -
-          </div>
-          <div class="label grid__col grid__col--2-of-8">
-            Broadcast
-          </div>
-          <div id="broadcast" class="text grid__col grid__col--6-of-8">
-            -
-          </div>
-          <div class="label grid__col grid__col--2-of-8">
-            Gateway
-          </div>
-          <div id="gateway" class="text grid__col grid__col--6-of-8">
-            -
-          </div>
-          <div class="label grid__col grid__col--2-of-8">
-            mask
-          </div>
-          <div id="mask" class="text grid__col grid__col--6-of-8">
-            -
-          </div>
           <div class="label grid__col grid__col--2-of-8">
             mode
           </div>
           <div id="mode" class="text grid__col grid__col--6-of-8">
             -
           </div>
+          <div id="ipSettings" style="display: none">
+            <div class="label grid__col grid__col--2-of-8">
+              IP Address
+            </div>
+            <div id="ip" class="text grid__col grid__col--6-of-8">
+              -
+            </div>
+            <div class="label grid__col grid__col--2-of-8">
+              Broadcast
+            </div>
+            <div id="broadcast" class="text grid__col grid__col--6-of-8">
+              -
+            </div>
+            <div class="label grid__col grid__col--2-of-8">
+              Gateway
+            </div>
+            <div id="gateway" class="text grid__col grid__col--6-of-8">
+              -
+            </div>
+            <div class="label grid__col grid__col--2-of-8">
+              mask
+            </div>
+            <div id="mask" class="text grid__col grid__col--6-of-8">
+              -
+            </div>
+          </div>
 
           <div id="syncLabel" class="label grid__col grid__col--2-of-8">
             Control
           </div>
           <div class="text grid__col grid__col--6-of-8">
             <button type="button" id="reload" onclick="">Reload</button>
             <button type="button" id="request" onclick="">Request</button>
-            <button type="button" id="assign" onclick="">Assign</button>
+            <!-- <button type="button" id="assign" onclick="">Assign</button> -->
```

Comment:
Did you intend to comment this so we can re-enable this button at one point?

Reply:
Yep!